pkg/worker/builder.go

```diff
@@ -116,3 +133,16 @@ func filterStages(stages []wharfyml.Stage, nameFilter string) []wharfyml.Stage {
 	}
 	return result
 }
+
+func shouldNotRunStage(anyStageFailed bool, s wharfyml.Stage) bool {
```

Comment:
Personal preference: I'm generally not a fan of bool functions that return a "negative" expression. Consider inverting this

Reply:
Changed name to `ShouldSkip` and made it a method of the `wharfyml.Stage` struct instead, and added tests.

So far we've been trying to avoid inverting in if-statements in this code base.
If something is always inverted, or mostly, then renaming it has been preferred. My previous choice of name here, `shouldNotRunStage` was very poor though. Hope the new name is acceptable :)